cmd/monitoring/main.go

```diff
@@ -5,12 +5,18 @@ import (
 
 	"github.com/gagliardetto/solana-go/rpc"
 	relayMonitoring "github.com/smartcontractkit/chainlink-relay/pkg/monitoring"
-	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring"
 	"github.com/smartcontractkit/chainlink/core/logger"
+
+	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring"
 )
 
 func main() {
-	coreLog := logger.NewLogger()
+	coreLog, closeLggr := logger.NewLogger()
+	defer func() {
+		if closeLggr != nil {
+			_ = closeLggr()
```

Comment:
Why do we throw away the error? If this errors we probably want to log it e.g. with `log.Println` or `log.Error`

Reply:
Fixed